packages/react-components/react-tree/src/components/Tree/useTree.ts

```diff
@@ -78,16 +80,17 @@ function useRootTree(props: TreeProps, ref: React.Ref<HTMLElement>): TreeState {
   const { appearance = 'subtle', size = 'medium' } = props;
 
   const { targetDocument } = useFluent_unstable();
-  const { openItems: stateOpenSubtrees, defaultOpenItems: defaultOpenSubtrees, onOpenChange } = props;
-  const [openItems, setOpenSubtrees] = useControllableState({
-    state: React.useMemo(() => normalizeOpenSubtreesOrUndefined(stateOpenSubtrees), [stateOpenSubtrees]),
-    defaultState: () => normalizeOpenSubtrees(defaultOpenSubtrees),
-    initialState: [],
+  const [openItems, setOpenItems] = useControllableState({
+    state: React.useMemo(() => props.openItems && ImmutableSet.from(props.openItems), [props.openItems]),
+    defaultState: React.useMemo(() => props.defaultOpenItems && ImmutableSet.from(props.defaultOpenItems), [
```

Comment:
No, since the condition on this case is not a boolean, `props.openItems` is either an `Iterable` or `undefined`, and the same goes to `props.defaultOpenItems`